spec/graphql/resolvers/billable_metrics_resolver_spec.rb

```diff
@@ -38,9 +36,12 @@
     aggregate_failures do
       expect(result['data']['billableMetrics']['collection'].count).to eq(organization.billable_metrics.count)
       expect(result['data']['billableMetrics']['collection'].first['id']).to eq(metric.id)
-      expect(result['data']['billableMetrics']['collection'].first['flatGroups']).to eq(
-        [{ 'id' => group2.id, 'key' => 'aws', 'value' => 'usa' }],
-      )
+
+      expect(result['data']['billableMetrics']['collection'].first['filters'].first['key']).to eq(filter1.key)
+      expect(result['data']['billableMetrics']['collection'].first['filters'].first['values']).to match_array(filter1.values)
+
+      expect(result['data']['billableMetrics']['collection'].first['filters'].second['key']).to eq(filter2.key)
+      expect(result['data']['billableMetrics']['collection'].first['filters'].second['values']).to match_array(filter2.values)
```

Comment:
Layout/LineLength: Line is too long. [126/120] (https://rubystyle.guide#max-line-length)